src/pages/app-development/index.md

```diff
@@ -1,6 +1,66 @@
 ---
-title: Adobe Commerce App Developer Guide Overview
-description: 
+title: Adobe Commerce App Developer's Guide Overview
+description: Learn how to make the transition from developing Commerce PHP extensions to developing Out-of-Process apps.
 ---
 
-# Adobe Commerce App Developer Guide Overview
+# Adobe Commerce App Developer's Guide Overview
+
+Developers have traditionally used PHP to create in-process extensions that add features, functionality, services, and integrations to Adobe Commerce. This model requires any new code to be compatible with upgrades, the server's PHP version, and many other essential server applications and services that Commerce uses.
+
+Adobe Developer App Builder uses out-of-process extensibility to avoid these compatibility issues. It provides a unified third-party extensibility framework for integrating and creating custom applications that extend Adobe Commerce. Since this extensibility framework is built on Adobe's infrastructure, developers can build custom microservices, and extend and integrate Adobe Commerce across other Adobe solutions and third-party integrations.
```

Comment:
I know this is not a marketing website, but out-of-process extensibility, it's something that we speak of internally and haven't really seen it as advertised or used as a standard industry topic. Might be alright, though

Also build custom microservices, that is something that we do advertise, where in reality they are functions ( lambda like), or some they can be bigger services like Catalog Service (we call them internally services because they are way bigger than a microservice with a complex API and functionality)
The only similarity between those is that they use the same mechanisms of a distributed system to communicate and share data and extend: web API and messagebus for events. - with tools provided by app builder